package.json

```diff
@@ -55,7 +57,10 @@
     "react-redux": "^5.0.7",
     "redux": "^4.0.0",
     "rimraf": "^2.6.2",
+    "sass-loader": "^7.0.1",
+    "scss-loader": "^0.0.1",
```

Comment:
Why do we need `scss-loader`? The webpack config doesn't use this package, but rather the `sass-loader`.

The yarn lockfile should be affected as well.

Reply:
I just checked and the package consists of an emtpy `package.json`. We should remove this dependency.